docs/en/latest/how-to-build.md

```diff
@@ -31,6 +31,20 @@ So before installation, please follow the different operating systems [install D
 
 You can install Apache APISIX in a variety of ways, including source code packages, Docker, and Luarocks.
 
+### Installation via RPM package (CentOS 7)
+
+```shell
```

Comment:
I see we use `bash` and `shell`, do we need to use the same lang?

Reply:
The original one used `shell`. Does it really matter to use only `bash` or `shell`? I don't want to waste time on a trivial thing.

Reply:
No, just want to confirm.

Reply:
I see.